lib/Dist/Zilla/App/Command/build.pm

```diff
@@ -16,6 +16,38 @@ does all the things required to build your distribution.  By default, it will
 also archive your distribution and leave you with a complete, ready-to-release
 distribution tarball.
 
+To go a bit further in depth, the C<build> command will do two things:
+
+=over
+
+=item
+
+Generate a directory containing your module, C<Foo-0.100>. This directory is
+complete. You could create a gzipped tarball from this directory and upload it
+directly to C<PAUSE> if you so desired. You could C<cd> into this directory and
+test your module so you're certain everything is as it should be before
+uploading to C<PAUSE>.
```

Comment:
I don't think we should suggest doing this, as it can cause files such as MYMETA.json to be left behind in the tarball, which should never appear in an uploaded distribution.  (And presently, even doing another 'build' afterward won't actually wipe out what was there before -- see issue #521 -- one would have to explicitly delete the directory, or run `dzil clean`).  `dzil test` is much safer for testing the distribution.